src/entity/terrain_mesh.hpp

```diff
@@ -22,23 +22,21 @@ class TerrainMesh : public MeshLoader::SingleComplexMesh {
     unsigned int num_vertices_;
 
  public:
-    inline TerrainMesh(const TerrainMesh& obj) {
-        vertex_buffer_ = obj.vertex_buffer_;
-        color_buffer_ = obj.color_buffer_;
-        normal_buffer_ = obj.normal_buffer_;
-        element_buffer_ = obj.element_buffer_;
-        num_vertices_ = obj.num_vertices_;
-        color_texture_ = obj.color_texture_;
-    };
-
-    inline TerrainMesh(const StaticMesh& obj) {
-        vertex_buffer_ = obj.get_vertex_buffer();
-        color_buffer_ = obj.get_color_buffer();
-        normal_buffer_ = obj.get_normal_buffer();
-        element_buffer_ = obj.get_element_buffer();
-        num_vertices_ = obj.get_num_vertices();
-        color_texture_ = obj.get_color_texture();
-    };
+    inline TerrainMesh(const TerrainMesh& obj) :
+        vertex_buffer_(obj.get_vertex_buffer()),
+        color_buffer_(obj.get_color_buffer()),
+        normal_buffer_(obj.get_normal_buffer()),
+        element_buffer_(obj.get_element_buffer()),
+        color_texture_(obj.get_color_texture()),
+        num_vertices_(obj.get_num_vertices()){};
+
+    inline TerrainMesh(const StaticMesh& obj) :
+        vertex_buffer_(obj.get_vertex_buffer()),
+        color_buffer_(obj.get_color_buffer()),
+        normal_buffer_(obj.get_normal_buffer()),
+        element_buffer_(obj.get_element_buffer()),
+        color_texture_(obj.get_color_texture()),
+        num_vertices_(obj.get_num_vertices()){};
```

Comment:
FORMAT

Reply:
Formatting does not make it look nice. It puts two initializations on the same line. That is the only change.